src/WebJobs.Script/Description/FunctionMetadata.cs

```diff
@@ -59,6 +59,8 @@ public FunctionMetadata()
         /// </summary>
         public bool IsDirect { get; set; }
 
+        public string FunctionId { get; set; } = Guid.NewGuid().ToString();
```

Comment:
>public string FunctionId  [](start = 8, length = 25)

What is this? What are the semantics of it?  (Why does it need a new guid every time? )

Reply:
This doesn't create a new one each time right - it is initialized once on construction? Anyhow I'm also wondering why we need this - we already have an InvocationId that we use everywhere.